content/sensu-go/6.1/operations/maintain-sensu/troubleshoot.md

```diff
@@ -12,6 +12,50 @@ menu:
     parent: maintain-sensu
 ---
 
+## A word about etcd and disk latency
```

Comment:
@hillaryfraley I think the heading and TL;DR bits here are maybe a little off-brand for us. LMK your thoughts on how to frame this kind of advice?

Reply:
@cwjohnston I think you might have removed this content altogether, is that right? I can't see it in the current commit for this PR.